tests/baselines/reference/decoratedClassExportsCommonJS1.js

```diff
@@ -24,5 +24,5 @@ let Testing123 = Testing123_1 = class Testing123 {
 exports.Testing123 = Testing123;
 Testing123.prop1 = Testing123_1.prop0;
 exports.Testing123 = Testing123 = Testing123_1 = __decorate([
-    Something({ v: () => Testing123_1 })
+    Something({ v: () => Testing123 })

```

Comment:
I don't understand how this diff happened. I assume it's because of the first block in the checker getting deleted, but I thought the edit to the second block would cause the reference to still get generated as Testing123_1.

Reply:
`Testing123_1` represents the class binding *inside* the class body, however a class decorator is evaluated *outside* of the class body, so it was pointing to the wrong reference prior to this change.